package.json

```diff
@@ -5,10 +5,13 @@
   "main": "build/userale-2.1.1.js",
   "scripts": {
     "pretest": "gulp lint",
-    "test": "gulp test",
+    "test": "gulp test && npm run journey:ci",
     "build": "gulp build",
     "clean": "gulp clean",
     "dev": "gulp dev",
+    "journey": "cypress run",
+    "journey:debug": "cypress open",
+    "journey:ci": "start-server-and-test 'node example/server.js' 8000 journey",
```

Comment:
Got it, I think that's because I was using the global Node binary on my machine to start the `server.js` server, the "node" in `node example/server.js`. Your suggested change is the way to go though, thanks for the catch!